src/elements/content-picker/selectionCellRenderer.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @flow
+ * @file Function to render the checkbox table cell
+ * @author Box
+ */
+
+import React from 'react';
+import Checkbox from '../../components/checkbox';
+import RadioButton from '../../components/radio/RadioButton';
+import isRowSelectable from './cellRendererHelper';
+
+export default (
+    onItemSelect: Function,
+    selectableType: string,
+    extensionsWhitelist: string[],
+    hasHitSelectionLimit: boolean,
+    isRadio: boolean,
+): Function => ({ rowData }: { rowData: BoxItem }) => {
```

Comment:
minor: we are trying to move away from using types `Function`, `any`, and `Object`. You can use `// flow strict ` to help you with the assertions.

Reply:
LGTM otherwise!